### ISSUE TITLE
Please remove stolen intellectual property

### ISSUE THREAD

**Issue by leafo** [w3itch-crafter/w3itch-frontend#340]
You have stolen the UI of itch.io, blatantly ripping it off, in addition to using our name directly in your project. As far as I'm concerned, this is scam to trick people by using @itchio's established brand on an unrelated project for your own profit.

I request that you immediately remove any likeness, including our name, from your project.

Additionally, upon going through the content on your platform, it appears you have stolen many games (see rpgmaker.net) for reupload without the consent of the creators of the games. For the sake of respecting creators and their work, I request that you remove stolen content from your platform.

**Comment by quou** [comment]
A poorly designed UI is better than a stolen one.

I think you've done it with the hope of having people confuse your site with itch.io and piggyback off of itch.io's userbase. You could have used a pre-designed website template. It's a sorry excuse to say you have no designers.

**Comment by Garfield550** [comment]
@quou We will communicate with the project manager about looking for new designs, we have no right to do anything until then.

**Comment by moburkhardt** [comment]
you have no right to steal designs either

**Comment by Garfield550** [comment]
@iSaluki Do whatever you want, the domain owner is not me.

**Comment by iSaluki** [comment]
That doesn't make it right that this project exists and does what it is doing. Just because you aren't the domain owner does not mean that you have no responsibility for this as a contributor.

**Comment by Lothrazar** [comment]
"we have no right to do anything until then."

You had no right to rip everyone off but that didnt stop you.  Whats stopping you now from taking the site down?

You have no right to do the right thing @Garfield550

**Comment by iSaluki** [comment]
I don't see how this is a totalitarian society, and I don't see how that would impact your ability to remove IP violations. All people are asking you to do is remove content you do not own the rights to. Is that really that unreasonable?

**Comment by iSaluki** [comment]
I understand, thanks for letting us know. Could you update people here when something happens?

**Comment by Garfield550** [comment]
@iSaluki I will do my best.

**Comment by iSaluki** [comment]
Thank you

**Comment by Garfield550** [comment]
<img width="1097" alt="image" src="https://user-images.githubusercontent.com/3471836/176819787-e864ce34-890d-4338-9a7f-f5c47a86a5bb.png">

 Update: All games on the site have now been removed.

**Comment by joshsleeper** [comment]
@Garfield550 the request of this issue was to remove all likeness to itch.io, which as far as I can tell still remains. Please reopen this issue to continue tracking this work.

**Comment by quou** [comment]
@joshsleeper The site no longer exists, only the GitHub repositories remain.

**Comment by joshsleeper** [comment]
That's better although if the repositories contain itch.io source as they did previously that also needs to be removed to be compliant with the issue request, and likely itch.io source code licensing

**Comment by Garfield550** [comment]
@joshsleeper Our project manager @lychees are recruiting designers, once the design is complete, we will open a new Issue to track it.

**Comment by joshsleeper** [comment]
I appreciate the intent here, really, but that doesn't change the fact that the request for this issue was to remove the design and content stolen from itch.io, which I believe still hasn't been completed.

Until that's done you should really leave this issue open for tracking and accountability, especially since it was literally opened by the creator of itch.io.

Closing this story and opening a new one for the same thing phrased differently serves no purpose other than adding a layer of abstraction to confuse the possibly hundreds of people trying to follow the progress and ensure this is properly resolved.

Feel free to rename the issue to be something your team finds more appropriately actionable if you must, but please reopen this story and actually use it for updates.

**Comment by Garfield550** [comment]
@lychees Please reopen this issue.

**Comment by BenoitAnastay** [comment]
The test server is still running

**Comment by BStrauss3** [comment]
If itch is YOUR copyrighted property, file a DMCA Takedown notice with GitHib
